### ISSUE TITLE
3.3.0: tests.test_version.VersionTestCase fails

### ISSUE THREAD

**Issue by mtelka** [FactoryBoy/factory_boy#1036]
I'm runnig tests for factory_boy 3.3.0 and the following test fails:
```
FAIL: test_version (tests.test_version.VersionTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "$(BUILD_DIR)/tests/test_version.py", line 10, in test_version
    self.assertEqual(factory.__version__, "3.2.1.dev0")
AssertionError: '3.3.0' != '3.2.1.dev0'
- 3.3.0
+ 3.2.1.dev0
```

**Comment by francoisfreitag** [comment]
That looks like an oversight. The version for 3.3.0 should indeed be 3.3.0 and not 3.2.1.dev0. That test can be ignored for the published release, I’ll remove it on the master branch.

**Comment by francoisfreitag** [comment]
I see that @rbarrois fixed the issue in f0859075b048b85c8ed44c051f18070cbd20d03d.
